### DISCUSSION TITLE
Welcome to GO-BlockChain-Simulation Discussions!

### DISCUSSION THREAD

**Discussion by SadikSunbul** [SadikSunbul/GO-BlockChain-Simulation#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cooltya** [comment]
I am getting an error about the badger in part-3 and part-4, it says there is a problem with the logs. What should I do

**Reply by SadikSunbul** [reply]
Delete the tmp/blocks file and then create the block file under tmp. In the project location type "go run main.go -help" first "go run main.go createblockchain -address "pubKey" " Create a blockchain by running this and then you can do the other steps.
